src/passes/transform/isolate_primitives.cpp

```diff
@@ -0,0 +1,84 @@
+#include "coreir/passes/transform/isolate_primitives.h"
+
+std::string CoreIR::Passes::IsolatePrimitives::ID = "isolate_primitives";
```

Comment:
can just put this inside the namespace

Reply:
This is consistent with all the other passes. A PR to change all of them would be better.